content/special-topics/color-management/overview.md

```diff
@@ -12,4 +12,4 @@ author: "people"
 
 - darktable automatically reads the display profile of your monitor (if properly configured) for accurate on-screen color rendition. Multi-screen setups are fully supported as long as a system service like colord is in place and properly set up to inform darktable of the correct monitor profile.
 
-- Output files can be encoded in one of darktable's built-in profiles (e.g. sRGB or AdobeRGB) or into any other output color space that the supplied by the user as an ICC profile.
+- Output files can be encoded in one of darktable's built-in profiles, including sRGB, Adobe, ProPhoto, Rec. 2020 and Rec. 709 standard RGB color spaces, or into a color space supplied by the user as an ICC profile.
```

Comment:
There's no such space as "Adobe", it should be "AdobeRGB". This together with sRGB are the most common use cases for output profile, so I don't think it is worth trying to list some of the others unless it is going to be done comprehensively. We can probably assume that if someone wants to use one of those other color spaces, they are an advanced user, and can see in the drop-down list what their options are.

Reply:
Fine with me, I only added them to the orginal PR at the request of Pascal Obry.

Reply:
But there's still that typo in the last sentence to fix. ;-)